packages/frame-host/src/endpoint.ts

```diff
@@ -0,0 +1,81 @@
+import * as Comlink from "comlink";
```

Comment:
yea it helped me quickly prototype though I think it'd be nice to remove the dependency for the 1.0 release—I think it could be replaced with one relatively small module